src/app/common/get-error-message.ts

```diff
@@ -3,20 +3,59 @@ import { TxBroadcastResultRejected } from '@stacks/transactions';
 export function getErrorMessage(
```

Comment:
Not 100% sure my return messages are what we want to show here, so maybe let me know changes and/or if we should just show some as the default message. I did make sure they are all accounted from here now.

In addition to the api, ref: https://github.com/stacks-network/stacks-blockchain/blob/master/src/chainstate/stacks/db/blocks.rs#L259